### ISSUE TITLE
Remove Root Element XML Namespace Attributes

### ISSUE THREAD

**Issue by Pudgie** [sissaschool/xmlschema#120]
Hi,

First of all, I just want to say thank you for adding the strip_namespaces in the newest release! It's extremely helpful. I want to ask if it's also possible to remove XML namespace (xmlns) attributes from the root element while keeping other attributes when converting xml to json. 

For example:
```
<?xml version="1.0" encoding="UTF-8"?>
<SomeRoot xmlns:ns1="namespace-xsd">
   <ns1:Container CreateDate="2019-06-29"/>
</SomeRoot>
```

The resulting xmlschema.to_json with strip_namespaces and preserve_root enabled in the default xml converter is this:
```
{
    "SomeRoot": {
         "xmlns:ns1": "namespace-xsd",
         "Container": {
              "CreateDate": "2019-06-29"
         }
    }
}
```
Is it possible to remove _"xmlns:ns1": "namespace-xsd"_ completely without having to strip the json afterwards?

Thanks!

**Comment by brunato** [comment]
Yes, this is coherent with your previous request. Adding a pair of additional checks of `strip_namespaces` on the three converter classes that include namespace information are not a problem. Will be available with the next release.
Bye

**Comment by brunato** [comment]
The additional option *strip_namespace* is implemented in release v1.0.14.
Bye
